Makefile

```diff
@@ -5,16 +5,16 @@ style:
 	black --preview $(check_dirs)
 	isort $(check_dirs)
 
-BATCH_SIZE ?= 1
-DTYPE ?= float16
+BATCH_SIZE ?= 4
+DTYPE ?= float32
 HIDDEN_SIZE ?= 2048
 N_HEAD ?= 16
 N_LAYER ?= 24
 N_POSITION ?= 2048
-MAX_INPUT_LENGTH ?= -1
+MAX_INPUT_LENGTH ?= 512
 
-RUN_HF := python3 src/main.py --pipeline_class=HF_Pipeline
-RUN_DS := deepspeed --num_gpus 1 src/main.py --pipeline_class=DS_Pipeline
+RUN_HF := python -m src.main --pipeline_class=HF_Pipeline
```

Comment:
It's not an installed package so we should refer to it by path, not module

Reply:
```python
python3 src/main.py --pipeline_class=ONNX_Pipeline --pretrained_model=mayank31398/santacoder --tokenizer=mayank31398/santacoder --trust_remote_code --dtype=float16 --batch_size=1 --max_input_length=-1 device=cuda
Traceback (most recent call last):
  File "src/main.py", line 11, in <module>
    from src.metrics import Metrics
ModuleNotFoundError: No module named 'src'
make: *** [Makefile:73: santacoder-onnx] Error 1
```
For me it doesn't work :)
are you pip installing it?

Reply:
Ill need to convert everything in the code to use relative paths to make this work I guess?

Reply:
Hmm I had another look, I guess you're right. The `-m` is needed to do the imports properly (though it still needs to be called from the repo root), in the dockerfile I'm just adding to the pythonpath. Relative paths wouldn't work either.

Please keep `python3` as the executable, `python` tends to refer to python2